components/Footer.js

```diff
@@ -16,24 +17,28 @@ export default function Footer() {
       <div className="max-w-screen-xl mx-auto py-12 px-4 overflow-hidden sm:px-6 lg:px-8">
         <nav className="-mx-5 -my-2 flex flex-wrap justify-center">
           <div className="px-5 py-2">
-            <a href="/about" className="text-base leading-6 text-gray-500 hover:text-gray-900">
-              About
-            </a>
+          <div className="text-base leading-6 text-gray-500 hover:text-gray-900">
+            <Link href="/about" className="text-base leading-6 text-gray-500 hover:text-gray-900">
+             <a> About </a>
```

Comment:
Inside `<Link>` component we can pass `<a>` HTML Tag, but if you don't have any "additional" attributes in that tag, you should use only string as children:

```suggestion
              About
```

Please remove any `<a>` occurrences inside `Link` , or maybe I don't know about something that NextJS handles and you know, so please describe that :)

Reply:
You can check how NextJS handled string inside `Link` -> https://github.com/vercel/next.js/blob/canary/packages/next/client/link.tsx#L226

Reply:
Sure, I will sure add `nofollow` in every `rel` present in code. 
On the discussion of `<a>` tag inside `<Link>` component, it is also recommended to use it, as by next js documentation.  Here is the [link](https://nextjs.org/docs/api-reference/next/link). Reason: Using only `<Link>` component doesn't give a cursor change to the pointer while hovering over the link. Hence, `<a>` tag is required.

Reply:
Ok, thanks for explanation! I don't know about that :)

Reply:
@magicmatatjahu I have resolved the issues. You can surely check now.